### PR TITLE
add yarn audit guidance (fixes #99)

### LINKED ISSUE #99: Add docs about running yarn audit as part of a PR
Update docs on how to add an audit to a PR that changes dependencies

### PR DESCRIPTION
**Pull request by sojeri**
This should close #99. It adds dangerjs checks for plain text yarn audit contents.

I've also updated the docs to reference our use of dangerjs and to fix some drift between the table of contents and the actual contents.

### CODE REVIEW

File: dangerfile.ts [hunk 1]
```diff
@@ -32,15 +32,30 @@ if (packageChanged && !lockfileChanged) {
...
+// Ensure we have access to github for these checks
 let isTrivial = false;
+let isYarnAuditMissing = false;
 if (danger.github) {
-  isTrivial = includes((danger.github.pr.body + danger.github.pr.title), "#trivial")
+  let prBody = danger.github.pr.body
+  
+  isTrivial = includes((prBody + danger.github.pr.title), '#trivial')
+    
+  if (lockfileChanged) {
```
Comment (suzubara) [inline dangerfile.ts:43]: I wonder if this will be the case for all dependabot PRs.. we should probably skip all checks if the user is dependabot, if you want to add that here or we can do it in another PR

**Comment by trussworks-infra** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please add a changelog entry for your changes.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6fdc8b9e69a8e39245147981f586d0d65277505e
</p>
